core/src/hlist.rs

```diff
@@ -127,7 +137,7 @@ pub trait HList: Sized {
     /// assert_eq!(<HList![i32, bool, f32]>::static_len(), 3);
     /// # }
     /// ```
-    #[deprecated(since = "0.1.31", note = "Please use LEN instead")]
+    #[deprecated(since = "0.1.31", note = "Please use Len::USIZE instead")]
```

Comment:
On the other hand, I think we should probably think about finally removing this long-deprecated method..